plugins/woocommerce/includes/wc-formatting-functions.php

```diff
@@ -1272,7 +1272,10 @@ function wc_format_stock_quantity_for_display( $stock_quantity, $product ) {
  * @return string
  */
 function wc_format_sale_price( $regular_price, $sale_price ) {
-	$price = '<del aria-hidden="true">' . ( is_numeric( $regular_price ) ? wc_price( $regular_price ) : $regular_price ) . '</del> <ins>' . ( is_numeric( $sale_price ) ? wc_price( $sale_price ) : $sale_price ) . '</ins>';
+	$price  = '<span class="screen-reader-text">' . __( 'Previous price:', 'woocommerce' ) . '</span>';
```

Comment:
Same markup as in https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/base/components/product-price/index.tsx